Makefile

```diff
@@ -23,6 +23,7 @@ BUILDBRANCH      := $(shell ./scripts/compute_branch.sh)
 CHANNEL          ?= $(shell ./scripts/compute_branch_channel.sh $(BUILDBRANCH))
 DEFAULTNETWORK   ?= $(shell ./scripts/compute_branch_network.sh $(BUILDBRANCH))
 DEFAULT_DEADLOCK ?= $(shell ./scripts/compute_branch_deadlock_default.sh $(BUILDBRANCH))
+export GOCACHE=$(SRCPATH)/tmp/go-cache
```

Comment:
I'm not sure that this is even needed. but if it does, the "touch" was somehow disappeared in this refactoring.

Reply:
Thanks I just added it back. It was a while ago, but I think the touch was needed because of some sort of issue with go accessing empty folders on virtualized disks in docker containers.